### PR TITLE
chore: add keysExist/keyExists

### PR DESCRIPTION
**Pull request by poppoerika**
Added `keysExist` and `keysExist`.

Ticket: https://github.com/momentohq/dev-eco-issue-tracker/issues/211

### CODE REVIEW

File: docs/develop/api-reference/index.mdx [hunk 1]
```diff
@@ -485,6 +485,46 @@ The resulting incremented value must be between -9223372036854775808 and 9223372
 
 :::
 
+### keysExists
```
Comment (anp13) [inline docs/develop/api-reference/index.mdx:488]: nit: `keyExists`

File: docs/develop/api-reference/index.mdx [hunk 2]
```diff
@@ -485,6 +485,46 @@ The resulting incremented value must be between -9223372036854775808 and 9223372
...
+
+Checks if a provided key exist in the cache.
```
Comment (anp13) [inline docs/develop/api-reference/index.mdx:490]: nit: Checks if a provided key exists in the cache.

File: docs/develop/api-reference/index.mdx [hunk 3]
```diff
@@ -485,6 +485,46 @@ The resulting incremented value must be between -9223372036854775808 and 9223372
...
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| cacheName       | String          | Name of the cache.                            |
+| key        | String \| byte          | Key which is to be checked their existence in the cache.              |
```
Comment (anp13) [inline docs/develop/api-reference/index.mdx:495]: nit: Key which is to be checked for its existence in the cache.

File: docs/develop/api-reference/response-objects.md [hunk 4]
```diff
@@ -38,4 +38,4 @@ For fetch commands, the key or field exists in the cache and there is a "cache h
 
 ### Miss
 
-For fetch commands, the key or field does not exist and it a "cache miss."
\ No newline at end of file
+For fetch commands, the key or field does not exist and it a "cache miss."
```
Comment (anp13) [inline docs/develop/api-reference/response-objects.md:41]: nit: For fetch commands, the key or field does not exist in the cache and there is a "cache miss."

File: docs/develop/api-reference/index.mdx [hunk 5]
```diff
@@ -485,6 +485,46 @@ The resulting incremented value must be between -9223372036854775808 and 9223372
...
+</details>
+
+### keysExist
+
+Checks if provided keys exist in the cache.
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| cacheName       | String          | Name of the cache.                            |
+| keys        | String[] \| byte[]          | Keys which are to be checked their existence in the cache.              |
```
Comment (anp13) [inline docs/develop/api-reference/index.mdx:515]: nit: Keys which are to be checked for their existence in the cache.

**Review by anp13 — COMMENTED** [review]
LGTM. Just super nit comments.

**Comment by poppoerika** [comment]
Closing this PR since this is included in this PR: https://github.com/momentohq/public-dev-docs/pull/131

**Pull request closed without merging**